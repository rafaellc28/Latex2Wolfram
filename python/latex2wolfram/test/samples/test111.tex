% Test transpose matrix
\begin{bmatrix}
  6 & 3\\
  1 & 8
\end{bmatrix}^{T}
