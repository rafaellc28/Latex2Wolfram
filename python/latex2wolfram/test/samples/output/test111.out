(transpose {{6, 3}, {1, 8}})
